src/test/regress/output/multi_agg_distinct.source

```diff
@@ -97,3 +97,67 @@ SELECT count(distinct o_orderkey) FROM orders;
 ERROR:  cannot compute aggregate (distinct)
 DETAIL:  table partitioning is unsuitable for aggregate (distinct)
 HINT:  You can load the hll extension from contrib packages and enable distinct approximations.
+-- Hash partitioned tables:
+CREATE TABLE lineitem_hash (
+	l_orderkey bigint not null,
+	l_partkey integer not null,
+	l_suppkey integer not null,
+	l_linenumber integer not null,
+	l_quantity decimal(15, 2) not null,
+	l_extendedprice decimal(15, 2) not null,
+	l_discount decimal(15, 2) not null,
+	l_tax decimal(15, 2) not null,
+	l_returnflag char(1) not null,
+	l_linestatus char(1) not null,
+	l_shipdate date not null,
+	l_commitdate date not null,
+	l_receiptdate date not null,
+	l_shipinstruct char(25) not null,
+	l_shipmode char(10) not null,
+	l_comment varchar(44) not null );
+SELECT master_create_distributed_table('lineitem_hash', 'l_orderkey', 'hash');
+ master_create_distributed_table 
+---------------------------------
+ 
+(1 row)
+
+SELECT master_create_worker_shards('lineitem_hash', 4, 1);
+ master_create_worker_shards 
+-----------------------------
+ 
+(1 row)
+
+\COPY lineitem_hash FROM '@abs_srcdir@/data/lineitem.1.data' with delimiter '|'
+\COPY lineitem_hash FROM '@abs_srcdir@/data/lineitem.2.data' with delimiter '|'
+-- aggregate(distinct) on partition column is allowed
+SELECT count(distinct l_orderkey) FROM lineitem_hash;
+ count 
+-------
+  2985
```

Comment:
@lithp Sooo, I'd expected to see the results were same with range partitioned table, but, they are not. In range partitioned table, this result is 2986. Same as for the other queries.

This could be a bug on our side. Could you check the followings to ensure whether is it a bug or not:
* Check that the two files have distinct rows. This is important for the correctness of the results on the range table case. 
* Load the same data to a regular postgres table, and compare the results. 

Is the problem clear for you as well?